language/stdlib/modules/LibraAccount.move

```diff
@@ -943,6 +943,43 @@ module LibraAccount {
         }
     }
 
+    // The epilogue is invoked at the end of transactions when the transaction is aborted during execution or during
+    // regular epilogue.
+    //
+    // It collects gas and bumps the sequence number
+    fun failure_epilogue<Token>(
```

Comment:
Could we extract the common logic into a private function (or a pair of private functions) and call it from both success and failure epilogues? E.g.,

```
success_epilogue() {
        let transaction_fee_amount = txn_gas_price * (txn_max_gas_units - gas_units_remaining);
        assert(balance_for(sender_balance) >= transaction_fee_amount)
        epilogue(transaction_fee_amount);
}

failure_epilogue() {
        let transaction_fee_amount = txn_gas_price * (txn_max_gas_units - gas_units_remaining);
        epilogue(transaction_fee_amount);
}
```